### PR TITLE
chore(FX-3731): Saved Alerts copy updates

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Chore**

This PR resolves [FX-3731]

### Description

The purpose of this pr is to update the Alert Flow copies in the app and adding a View Duplicate button in the Duplicate alert prompt, in order collectors to be able to view with one press of a button the duplicate alert.

#### Screenshots 
|Screenshots||||
|--|--|--|--|
|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 14 26|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 14 21|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 28 57|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 24 08|
|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 14 56|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 15 34|Simulator Screen Shot - iPhone 13 mini - 2022-02-09 at 17 34 34|

#### Video

Simulator Screen Recording - iPhone 13 mini - 2022-02-09 at 17 37 32

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- saved alert copy updates and add view duplicate alert button - gkartalis

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- 

</details>

[FX-3671]: https://artsyproduct.atlassian.net/browse/FX-3671?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FX-3731]: https://artsyproduct.atlassian.net/browse/FX-3731?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (saved alert copy updates and add view duplicate alert button - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d8919adb59d99eef6b0e149b4bf73cdbf74a218b
</p>

File: src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx [hunk 1]
```diff
@@ -121,6 +122,10 @@ export const SavedSearchAlertForm: React.FC<SavedSearchAlertFormProps> = (props)
             onReplacePress: () => {
               submitHandler(userAlertSettings, clearedAttributes)
             },
+            onViewDuplicatePress: () => {
+              goBack()
+              navigate(`/my-profile/saved-search-alerts/${duplicateSavedSearchId}`)
```
Comment (dimatretyak) [inline src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:127]: **question**: should we open the page within the `Search` tab or `My Profile`?

Comment (gkartalis) [reply src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:127]: Good question, will take it with design and update accordingly

Comment (dimatretyak) [reply src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:127]: @gkartalis is there any feedback?

Comment (gkartalis) [reply src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:127]: @dimatretyak yup! Got it on Friday afternoon and it is the following:

> it makes sense to have the “profile” tab open when we view the duplicate.
but when the user selects BACK in the duplicate alert. this should bring them back to the original alert creation. (which then lives back in the search tab)

I was looking into this and it is kinda complicated, cause when we use switchTab it is a navigation action and we cannot do both (switchTab along with navigate to). Now trying navigate with replace but again not sure how to override the backButton of the alert in profile.

Comment (gkartalis) [reply src/lib/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:127]: 🤘 Sounds good. Will open a thread about this in `#product-find-explore` and suggest to have this merged as it is!

**Comment by gkartalis** [comment]
Merging. Context [here](https://artsy.slack.com/archives/C9SATFLUU/p1644848918930199), We decided to leave it as it is ✌️

**Pull request merged**